src/optimize.js

```diff
@@ -52,8 +52,8 @@ function validateAndFix(opts) {
     .forEach((key) => opts.plugins.push(key));
 }
 
-export default function optimize(content, opts = {}) {
+export default function optimize(content, opts = {}, path) {
```

Comment:
putting a required param after an optional one is not ideal. can't the path already be passed in `opts`?

Reply:
I see your point about 'putting a required param after an optional one is not ideal.', I will make the change.

Since the svg path is resolved in the plugin, I guess it is safe to pass the path from the code. Still a valid concern since the docs states that the options are passed to svgo. I can add this if not in the options.